invesalius/gui/dialogs.py

```diff
@@ -3366,7 +3366,7 @@ def _init_gui(self):
         self.CenterOnScreen()
 
     def _create_selectable_label_text(self, text):
-        label = wx.TextCtrl(self, -1, style=wx.TE_READONLY)
+        label = wx.TextCtrl(self, -1, style=wx.TE_READONLY, size=(105, 22))
```

Comment:
To set the initial size try to use this https://github.com/invesalius/invesalius3/blob/cc72604a0f867b4d9f68f6e12e99b5b0fc58d09c/invesalius/gui/utils.py#L24

Reply:
I moved the macOS related change out of this branch.